src/lib.rs

```diff
@@ -7,20 +7,17 @@ mod value_state;
 pub use user::User;
 pub use value_state::ValueState;
 
-imports!();
+elrond_wasm::imports!();
 
-#[elrond_wasm_derive::contract(AttestationImpl)]
+#[elrond_wasm::contract]
 pub trait Attestation {
 	#[init]
-	fn init(&self, registration_cost: &BigUint, address: Address, max_nonce_diff: u64) {
-		self.set_registration_cost(registration_cost);
-		self.set_attestator_state(&address, &ValueState::Approved);
+	fn init(&self, registration_cost: Self::BigUint, address: Address, max_nonce_diff: u64) {
+		self.registration_cost().set(&registration_cost);
+		self.attestator_state(&address).set(&ValueState::Approved);
```

Comment:
should get var args of addresses. list of addresses.

Reply:
and should delete old list if exist

Reply:
Done.